RELEASE.md

```diff
@@ -6,30 +6,46 @@ Official Falcosecurity rules releases are automated using GitHub Actions. Each r
 
 In this repo, each ruleset is a standalone YAML file in the `/rules` directory (e.g. `falco_rules.yaml`, `application_rules.yaml`, ...). Each ruleset is released and versioned individually. When we release a ruleset, we do the following process:
 
-1. Determine a new version for the given ruleset (see the [section below](#versioning-a-ruleset))
-2. Create a new Git tag with the name convention `*name*-rules-*version*` (e.g. `falco-rules-0.1.0`, `application-rules-0.1.0`, ...). The naming convention is required due to this repository being a [monorepo](https://en.wikipedia.org/wiki/Monorepo) and in order to be machine-readable.
-3. A GitHub action will validate the repository [registry](./registry.yaml) and release the new OCI artifact in the packages of this repository
+1. Make sure that the `./github/FALCO_VERSIONS` file contains the most recent versions of Falco compatible with the given ruleset. The `master` version must not be used when releasing a ruleset.
+2. Determine a new version for the given ruleset (see the [section below](#versioning-a-ruleset))
+3. Create a new Git tag with the name convention `*name*-rules-*version*` (e.g. `falco-rules-0.1.0`, `application-rules-0.1.0`, ...). The naming convention is required due to this repository being a [monorepo](https://en.wikipedia.org/wiki/Monorepo) and in order to be machine-readable.
+4. A GitHub action will validate the repository [registry](./registry.yaml) and release the new OCI artifact in the packages of this repository
 
 ## Versioning a ruleset
 
 The version of the official Falco rulesets is compatible with [SemVer](https://semver.org/) and must be meaningful towards the changes in its structure and contents. To define a new version `x.y.z` for a given ruleset, consider the following guidelines. 
 
+Our automation will detect most of the following criteria and suggest a summary with all the changes relative to each of the three versioning categories (patch, minor, major). This provides a changelog and valuable suggestion on the next version to be assigned to each ruleset. However, be mindful that the versioning process cannot totally automated and always requires human attention (e.g. we can't automatically detect subtle semantic changes in rules). The automated jobs will use the versions of Falco defined in `./github/FALCO_VERSIONS` for validating and checking rulesets. The versions must be line-separated and ordered from the most recent to the least recent. The `master` keyword is valid and indicates the most recent dev version of Falco built from mainline, and must not be used when releasing a ruleset.
```

Comment:
Is `master` just a special case or do we support any valid git ref?

Reply:
It's a special case. Under the hood, that's the tag of the container image used for running Falco. As such, I think it's ok to document that the only safe cases available are `master` and the official falco version tags (e.g. `0.34.1`).

Reply:
Ok, so also, `latest` and `0.35.0-alpha5 should work, basically any [published tag](https://hub.docker.com/r/falcosecurity/falco/tags), shouldn't they? 

It's worth documenting that, IMO.

Reply:
Yep